### ISSUE TITLE
-dev configuration push endpoints are malformed

### ISSUE THREAD

**Issue by ncalexan** [mozilla-services/autopush#244]
Trouble at the old well!  The endpoints I'm getting back from -dev are malformed.  Looks like a host and port issue: `http://=:8082/push/gAAAAABWS-elV1hBCfCvB0MfP3fyRpKdpe7wlowK9wmiq2KcqUkdPsAP8xgxgGE-ljkXq41eTbWONLoJcGDMW7BYqx4ZfwmA8hH5rs4qa2qXzRd7fIlrQiI2z1xz3C2THy4Yg2Q-eQrAHDdsSSJwdHJp9JrPalOTfcw7jrYON0qIeLPzQY1gCac=`

Presumably that should be "https://autopush-dev.stage.mozaws.net/push/..."

**Comment by ncalexan** [comment]
@jrconlin over to you to configure -dev.

**Comment by jrconlin** [comment]
Jeremy?

It looks like the config file might have gotten goofed? Not sure why the
host is being set to "=".

**Comment by oremj** [comment]
Did a setting change? The autopush shared config has:

```
router_hostname = ec2-52-91-126-140.compute-1.amazonaws.com
router_port = 8081

endpoint_scheme = https
endpoint_hostname = updates-autopush-dev.dev.mozaws.net
endpoint_port = 443
```

**Comment by oremj** [comment]
Fixed in mozilla-services/puppet-config#1659
